### DISCUSSION TITLE
FastFlow

### DISCUSSION THREAD

**Discussion by brm738** [open-edge-platform/anomalib#159] [Feature Requests] [answered]
Big work. Congratulations!
Will you add the FastFlow algorithm to this benchmark?
https://arxiv.org/abs/2111.07677

**Comment by samet-akcay** [comment]
Thanks, we've been working on a WiP implementation. However, it's not ready yet. You could refer to the issues #114 and #133.

**Comment by ashwinvaidya17** [comment, marked as answer]
FastFlow was added to Anomalib in this PR https://github.com/openvinotoolkit/anomalib/pull/386
